nbs/core.ipynb

```diff
@@ -54,10 +54,15 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.            self._dataset_stored = False
</pre><p> </p><p>This line is not really needed, you can check if the dataset exists using <code>hasattr(self, 'dataset')</code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
done